src/phoenix/core/traces.py

```diff
@@ -273,16 +278,27 @@ def _process_span(self, span: pb.Span) -> None:
                 else max(self._max_start_time, start_time)
             )
         # Update cumulative values for span's ancestors.
-        for attribute_name, cumulative_attribute_name in (
-            (LLM_TOKEN_COUNT_TOTAL, ComputedAttributes.CUMULATIVE_LLM_TOKEN_COUNT_TOTAL.value),
-            (LLM_TOKEN_COUNT_PROMPT, ComputedAttributes.CUMULATIVE_LLM_TOKEN_COUNT_PROMPT.value),
+        counters_and_cumulative_attribute_names: List[Tuple[ReadableSpanValueCounter, str]] = [
             (
-                LLM_TOKEN_COUNT_COMPLETION,
+                lambda span: span[LLM_TOKEN_COUNT_TOTAL],
+                ComputedAttributes.CUMULATIVE_LLM_TOKEN_COUNT_TOTAL.value,
+            ),
+            (
+                lambda span: span[LLM_TOKEN_COUNT_PROMPT],
+                ComputedAttributes.CUMULATIVE_LLM_TOKEN_COUNT_PROMPT.value,
+            ),
+            (
+                lambda span: span[LLM_TOKEN_COUNT_COMPLETION],
                 ComputedAttributes.CUMULATIVE_LLM_TOKEN_COUNT_COMPLETION.value,
             ),
-        ):
-            existing_value = (existing_span[attribute_name] or 0) if existing_span else 0
-            new_value = new_span[attribute_name] or 0
+            (
+                lambda span: int(span.status.code == pb.Span.Status.Code.ERROR),
```

Comment:
I would recommend that we use `ERROR_COUNT` instead, as shown below, even though its value is always 1. Since the extra storage cost is negligible, I would recommend that we pay the cost in compute/storage instead of the cost of extra abstraction from using the lambdas, especially in such a straightforward situation.
```python
ERROR_COUNT = COMPUTED_PREFIX + "error_count"
```
```python
new_span[ERROR_COUNT] = int(span.status.code == pb.Span.Status.Code.ERROR)
```